### PR TITLE
Add min height to call composite media gallery

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What
* Ensure media gallery has a minimum height - below which the composite is not usable so this still fits in our responsive design
* Update the composite documentation to ensure it has a set height and width by default so users copying and pasting have a good experience
* Mention this issue in the call composite styling documentation

# Why
If the parent of the call composite doesn't have a height set, the video gallery was not visible.

# How Tested
In storybook
| before (if no height is set) | after (if no height is set) | default experience
| -- | -- | -- |
| image | image | image |

### CODE REVIEW

File: packages/react-composites/src/composites/CallComposite/styles/CallScreen.styles.ts [hunk 1]
```diff
@@ -30,27 +30,22 @@ export const callControlsContainer = mergeStyles({
...
   }
 };
 
 export const subContainerStyles: IStackStyles = {
   root: {
     overflow: 'hidden',
     width: '100%',
-    flexDirection: 'column',
-    display: 'flex',
-    flexBasis: '0'
```
Comment (PorterNan) [inline packages/react-composites/src/composites/CallComposite/styles/CallScreen.styles.ts:45]: Not sure if I remember this right, I remember set this flexBasis is something related to safari browser(it has the worst flex box support among all the browsers) compatibility, can we double check all the stylings are correct in safari/firefox browser?

**Review by PorterNan — APPROVED** [review]
Looks great, we finally have the min-height for the composite!

**Review by alcail — APPROVED** [review]

**Review by anjulgarg — APPROVED** [review]

File: packages/react-composites/src/composites/CallComposite/styles/CallScreen.styles.ts (hunk 1 above)
Comment (JamesBurnside) [reply packages/react-composites/src/composites/CallComposite/styles/CallScreen.styles.ts:45]: Good note, I will double check this

Comment (JamesBurnside) [reply packages/react-composites/src/composites/CallComposite/styles/CallScreen.styles.ts:45]: Yep looks to be good on safari
Removing fixed height:
image
default experience:
image

Comment (JamesBurnside) [reply packages/react-composites/src/composites/CallComposite/styles/CallScreen.styles.ts:45]: And firefox: 
image

**Pull request merged**